### PR TITLE
Add country state abbreviation transformation robustness evaluation

### PR DESCRIPTION
**Pull request by Yiwen-Shi**

### CODE REVIEW

**Review by RoyRin — APPROVED** [review]
Looks good, thanks for implementing these changes.

File: transformations/country_state_abbreviation_transformation/README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@ This transformation provides flexible options for Country/State abbreviation and
...
+```
+The original accuracy is 96.0 and the perturbed accuracy is 96.0.
+
+- Text summarization: 
+```
+python evaluate.py -t CountryStateAbbreviation -task TEXT_TO_TEXT_GENERATION
+```
+The original bleu is 14.88 and the perturbed bleu is 14.61.
+
+
```
Comment (kaustubhdhole) [inline transformations/country_state_abbreviation_transformation/README.md:42]: Just to confirm, have you used the models used [here](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/evaluation) i.e. from textattack ?

Comment (Yiwen-Shi) [reply transformations/country_state_abbreviation_transformation/README.md:42]: TEXT_CLASSIFICATION used model aychang/roberta-base-imdb on the test[:20%] split of the imdb dataset
TEXT_TO_TEXT_GENERATION used the model sshleifer/distilbart-xsum-12-6 on the test[:20%] split of the xsum dataset.

I can update the test classification performance with textattack models if needed. Thank you!

File: transformations/country_state_abbreviation_transformation/README.md [hunk 2]
```diff
@@ -25,6 +25,30 @@ This transformation provides flexible options for Country/State abbreviation and
...
+Benchmark results:
+
+- Sentiment analysis: 
+```
+python evaluate.py -t CountryStateAbbreviation -task "TEXT_CLASSIFICATION" -m "textattack/roberta-base-imdb" -d "imdb" -p 20
+```
+
+| Model  | Datasets | Performance  | 
+|---|---|---|
+| textattack/roberta-base-SST-2 | sst2 | 94.0->94.0 (0.0) |
```
Comment (kaustubhdhole) [inline transformations/country_state_abbreviation_transformation/README.md:37]: Just to confirm, there was no difference in performance?

Comment (Yiwen-Shi) [reply transformations/country_state_abbreviation_transformation/README.md:37]: Yes, for the classification tasks, there is no difference. For the text summarization task, the performance reduced on xsum dataset.

**Review by kaustubhdhole — APPROVED** [review]

**Pull request merged**